stable/mariadb/values.yaml

```diff
@@ -31,8 +31,20 @@ usePassword: true
 # mariadbDatabase:
 
 
+service:
 ## Kubernetes service type
```

Comment:
To pass CI this comment on like 35 needs to be indented 2 spaces. Or, it could be above `service:`.